test/tizen_capi/unittest_tizen_capi.cpp

```diff
@@ -968,6 +968,147 @@ TEST(nntrainer_capi_summary, summary_02_n) {
   EXPECT_EQ(status, ML_ERROR_NONE);
 }
 
+TEST(nntrainer_capi_nnmodel, get_input_output_dimension_01_p) {
```

Comment:
please create a couple more test cases:
1. ml_tensors_info_h can only support till 16 inputs/outputs. Check if the model has than 16 inputs, then how does this API handle it.
2. trying to get the input/output dimension with an empty model.
3. trying to get input/output dimension before compiling the model
4. trying to get input/output dimension after destroying the model
5. verify that the dimension object stays valid/invalid when the model is destroyed

Reply:
First of all I made a test cases you mentioned above, but since the unittest_tizen_capi is turn off in #1328 we can't check the testcase in the upstream/main right now. So I tested on my local branch before #1328 is merged.
For the case 1, we are not implemented multi input so can't check it right now. 
For the case 5, dimension object stays valid even the model is destroyed.

Reply:
great.